portfolio/src/main/webapp/maps.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <!-- STYLESHEETS -->
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+    <!-- css animation library -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.0.0/animate.min.css"
+    />
+    <style>
```

Comment:
Any reason you include these stylings here instead of your css file? Makes it harder to find where styling is in the application

Reply:
Applies to other styling defined in this file.

Reply:
You're right no reason to include them here, I had just done styling inline to play around with it and forgot to move them to my css file.